core/libp2p/muxer/yamux/yamuxed_connection.hpp

```diff
@@ -49,7 +49,8 @@ namespace libp2p::connection {
      */
     explicit YamuxedConnection(
         std::shared_ptr<SecureConnection> connection,
-        muxer::MuxedConnectionConfig config = {},
+        muxer::MuxedConnectionConfig config =
+            muxer::MuxedConnectionConfig::makeDefault(),
```

Comment:
I think now that default parameter is not appropriate here - connections are only to be created via muxer, and it always has a config to be inserted here. So, please, make the parameter non-default and the ctor non-explicit

Reply:
refactored